src/server/api/api.ts

```diff
@@ -67,10 +68,11 @@ export const websocketHandler: WebsocketRequestHandler = (ws, req) => {
 export const apiRouter = Router();
 
 apiRouter.get("/get-ids", (_, res) => {
-    return res.send({
-        // ids: ["10", "11"],
-        ids: tcpServer.getConnectedIds(),
-    });
+    const ids = tcpServer.getConnectedIds();
+    if (IS_DEVELOPMENT) {
+        ids.push("dummy-id-1", "dummy-id-2");
```

Comment:
do non-number IDs work with the rest of the system?

Reply:
I would certainly hope so